src/Rasterization/SVGRasterizer.php

```diff
@@ -243,7 +243,7 @@ public function getHeight()
     public function getScaleX()
     {
         if (!empty($this->viewBox)) {
-            return $this->getDocumentWidth() / $this->viewBox[2];
+            return $this->width / $this->viewBox[2];
```

Comment:
Yes, you're right about the widths being 2 different things but I've tested with many images before my commit and found the rasterizer width to correspond better to the desired result than the document width. Below's an example with images:
Svg: 
![image](https://cloud.githubusercontent.com/assets/3414935/23358571/d5d4af7e-fce2-11e6-827d-b473731d470c.png)

Render with document width/height : 
![image](https://cloud.githubusercontent.com/assets/3414935/23358471/61d3ad32-fce2-11e6-8086-071f27ecfa12.png)

Render with rasterizer width/height:
![image](https://cloud.githubusercontent.com/assets/3414935/23358487/6f1d516e-fce2-11e6-9172-ac360fe792fe.png)

I've tested many other images, not just this one. You can try and test too if you want.